src/front/app/components/globals/FormGroup.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Input from './Input';
+import Label from './Label';
+
+const classNames = require('classnames');
+
+const inputTypes = [
+  // Input Based
+  'text',
+  'email',
+  'password',
+  'number',
+  'date',
+  'datetime',
+  'datetime-local',
+  'search',
+  'tel',
+  'color',
+  'file',
+  // Select Based
+  'select',
+  // Text Area Based
+  'textarea',
+];
+
+class FormGroup extends Component {
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    label: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]).isRequired,
+    inputType: PropTypes.oneOf(inputTypes),
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]),
+    className: PropTypes.string,
+    inputClasses: PropTypes.string,
+    labelClasses: PropTypes.string,
+    selectChildren: PropTypes.oneOfType([
```

Comment:
Ideally, don't do this. Make a `Select` component which acts as a `selectFormGroup`. Whenever you see yourself adding something that's  a bit too "specific", like this prop, you can think of an entirely new component.